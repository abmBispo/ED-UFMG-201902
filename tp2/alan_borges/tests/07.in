5000 100 6
974 psvrdu
1951 ogdjei
2147 rridto
2088 fbhkbj
1901 jgdzvh
1099 yqufxh
2229 gmxzfq
1595 utqbol
1872 ohxhet
1244 eriwea
1592 oudvul
616 mefepv
1144 rrngxl
95 dloxco
2336 uyudto
1645 wxwjdq
857 jdimhw
1464 gnampr
1610 uhynwh
233 pbibyy
56 wakduj
688 ipdhim
1222 rwinco
1473 cvvkty
141 chbpke
1787 htyboq
441 bosvtn
1129 zxnvoe
2226 nfjhno
214 vbvhpp
2388 ppsptm
2453 sguysv
1590 vvwlml
2079 obmkvq
1109 nfzvle
1187 unaxbe
624 pvscug
1523 godwyr
1487 jrgfub
2440 fwmgid
2439 uofqhg
1048 mkivrm
41 vpscje
648 yuydlz
197 qhaqnz
1757 xawpcy
1440 yvlbag
2279 iqkqoy
2052 dcbckb
897 mnkxzw
1368 rujiwm
1003 xdoqbr
718 szdewd
1166 ivtutd
199 ooeefn
1280 zaumht
1135 xpethr
629 nqcwvh
1926 levwfl
728 mjfysk
329 hoorzy
438 eotynx
750 rcyjpw
473 qnzlin
624 jyhagu
812 lkehat
1814 mfxffj
1172 ofcdep
1528 pkofik
1505 enzqsq
1561 emgkzj
509 ibcqsu
577 mzdkxh
751 jofmmn
144 hzdvht
1830 teuwid
1779 zblvhq
93 ltomtl
644 unmmmh
866 hsiruo
971 uyhlxf
687 diweae
2044 stqcvj
1086 tkzgsy
160 ymfuse
578 ujrqfm
1572 voadkk
2297 fydyky
2163 xrqlyg
2426 qkipxu
964 bragig
1733 prjisn
963 wedeix
1741 houbql
2071 qzauri
1178 znrici
244 wjdvwu
1062 obdipj
525 fggbtf
1024 gcbfxv