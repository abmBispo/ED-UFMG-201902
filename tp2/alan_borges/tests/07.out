1 bosvtn 441
1 chbpke 141
1 dloxco 95
1 eotynx 438
1 fggbtf 525
1 hoorzy 329
1 hzdvht 144
1 ibcqsu 509
1 ltomtl 93
1 ooeefn 199
1 pbibyy 233
1 qhaqnz 197
1 qnzlin 473
1 vbvhpp 214
1 vpscje 41
1 wakduj 56
1 wjdvwu 244
1 ymfuse 160
2 jyhagu 624
2 mefepv 616
2 mzdkxh 577
2 nqcwvh 629
2 pvscug 624
2 ujrqfm 578
2 unmmmh 644
2 yuydlz 648
3 diweae 687
3 ipdhim 688
3 jofmmn 751
3 mjfysk 728
3 rcyjpw 750
3 szdewd 718
4 hsiruo 866
4 jdimhw 857
4 lkehat 812
4 mnkxzw 897
4 wedeix 963
5 bragig 964
5 gcbfxv 1024
5 psvrdu 974
5 uyhlxf 971
5 xdoqbr 1003
6 mkivrm 1048
6 obdipj 1062
6 tkzgsy 1086
6 yqufxh 1099
7 nfzvle 1109
7 rrngxl 1144
7 xpethr 1135
7 zxnvoe 1129
8 ivtutd 1166
8 ofcdep 1172
8 unaxbe 1187
8 znrici 1178
9 eriwea 1244
9 rwinco 1222
9 zaumht 1280
10 gnampr 1464
10 rujiwm 1368
10 yvlbag 1440
11 cvvkty 1473
11 enzqsq 1505
11 jrgfub 1487
12 emgkzj 1561
12 godwyr 1523
12 pkofik 1528
13 oudvul 1592
13 voadkk 1572
13 vvwlml 1590
14 uhynwh 1610
14 utqbol 1595
14 wxwjdq 1645
15 houbql 1741
15 prjisn 1733
16 xawpcy 1757
16 zblvhq 1779
17 htyboq 1787
17 mfxffj 1814
18 ohxhet 1872
18 teuwid 1830
19 jgdzvh 1901
19 levwfl 1926
20 ogdjei 1951
20 stqcvj 2044
21 dcbckb 2052
21 qzauri 2071
22 fbhkbj 2088
22 obmkvq 2079
23 rridto 2147
23 xrqlyg 2163
24 gmxzfq 2229
24 nfjhno 2226
25 fydyky 2297
25 iqkqoy 2279
26 ppsptm 2388
26 uyudto 2336
27 qkipxu 2426
27 uofqhg 2439
28 fwmgid 2440
28 sguysv 2453
